v0.11.0 Release Tour.ipynb

```diff
@@ -0,0 +1,229 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "portuguese-barrier",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%manim -qm -v WARNING ImplicitFunctionDemo\n",
+    "\n",
+    "class ImplicitFunctionDemo(Scene):\n",
+    "    def construct(self):\n",
+    "        plane = NumberPlane()\n",
+    "        curve = ImplicitFunction(\n",
+    "            lambda x, y: x * y**2 - x**2 * y - 2,\n",
+    "            color=YELLOW,\n",
+    "        )\n",
+    "        self.add(plane)\n",
+    "        self.play(Create(curve))\n",
+    "        self.wait()\n",
+    "        elliptic_curve = ImplicitFunction(\n",
+    "            lambda x, y: x**3 - x + 1 - y**2,\n",
+    "            color=RED,\n",
+    "        )\n",
+    "        self.play(Create(elliptic_curve))\n",
+    "        self.wait()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "hourly-aerospace",
+   "metadata": {},
+   "source": [
+    "\n",
+    "### [#1991](https://github.com/ManimCommunity/manim/pull/1991): Added support for boolean operations on `VMobject`s\n",
+    "\n",
+    "Manim can now compute the union, intersection, and difference of mobjects!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "tutorial-receptor",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%manim -qm -v WARNING BooleanOperationDemo\n",
+    "\n",
+    "class BooleanOperationDemo(Scene):\n",
+    "    def construct(self):\n",
+    "        c1 = Circle(radius=2, fill_opacity=0.5, color=YELLOW).shift(LEFT)\n",
+    "        c2 = Circle(radius=2, fill_opacity=0.5, color=BLUE).shift(RIGHT)\n",
+    "        circles = VGroup(c1, c2)\n",
+    "        self.add(circles.scale(0.5))\n",
+    "        grid = VGroup(\n",
+    "            Union(c1, c2, color=GREEN, fill_opacity=0.5),\n",
+    "            Intersection(c1, c2, color=GREEN, fill_opacity=0.5),\n",
+    "            Difference(c1, c2, color=GREEN, fill_opacity=0.5),\n",
+    "            Exclusion(c1, c2, color=GREEN, fill_opacity=0.5)\n",
+    "        ).arrange_in_grid(2, 2, buff=3)\n",
+    "        self.add(grid)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "shaped-decimal",
+   "metadata": {},
+   "source": [
+    "### [#2118](https://github.com/ManimCommunity/manim/pull/2118): Added 3D support for `ArrowVectorField` and `StreamLines`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "younger-latex",
+   "metadata": {},
+   "source": [
+    "When passing a `z_range` parameter to `ArrowVectorField` or `StreamLines`, the vector field will be rendered in three dimensions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dedicated-market",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%manim -qm -v WARNING --disable_caching ThreeDArrowVectorField\n",
+    "\n",
+    "class ThreeDArrowVectorField(ThreeDScene):\n",
```

Comment:
Maybe add a quick mention that this example might take a while to render?

Reply:
Good idea: af0e905